ci/pipeline-dev.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - get: src
         resource: pr-((git-branch))
         trigger: true
-      - set_pipeline: build-container
+      - set_pipeline: owasp-zap-task
```

Comment:
We should make sure the set pipeline tasks passes and is run before we run the deploy task to make pipeline adjustments easier in the future.